script/ci.sh

```diff
@@ -17,7 +17,7 @@ source ./script/before_install.sh
 
 if [ "$ALPAKA_CI_OS_NAME" = "Linux" ]
 then
-  ./script/docker_install.sh
+  source ./script/docker_install.sh
```

Comment:
I do not think that this works ;-)

Reply:
With #1109 install.sh and run.sh will be run within a single invocation of `docker run` which should make it much easier to keep settings between the invocations.